Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1428,6 +1428,13 @@ namespace xr
         return su.TryGetPlaneByID(id);
     }
 
+    bool System::Session::Frame::IsTracking() const
+    {
+        // TODO - Plumb tracking states from OpenXR. For now this will maintain the current behavior where
+        // BabylonJS assumes tracking is always available.
+        return true;
+    }
```

Comment:
@chrisfromwork @rgerd FYI

Reply:
can we get an issue filed for this work?

---
In reply to: [527098276](https://github.com/BabylonJS/BabylonNative/pull/504#discussion_r527098276) [](ancestors = 527098276)

Reply:
Issue filed ([here](https://github.com/BabylonJS/BabylonNative/issues/505)). I've updated the comment in code with a link to the issue as well.